extension/src/lib/extensionHolder.js

```diff
@@ -0,0 +1,58 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+fluid.defaults("gpii.chrome.extensionHolder", {
+    gradeNames: "fluid.modelComponent",
+    extensionId: "",
+    members: {
+        extensionInstance: null
+    },
+    model: {
+        extensionEnabled: undefined
+    },
+    invokers: {
+        setEnabled: {
+            funcName: "gpii.chrome.extensionHolder.setEnabled",
+            args: ["{that}", "{arguments}.1"]
+        },
+        getEnabled: {
+            funcName: "gpii.chrome.extensionHolder.getEnabled",
+            args: "{that}.extensionInstance"
+        }
```

Comment:
I win this bet: https://developer.chrome.com/extensions/management#event-onEnabled